department-of-veterans-affairs/va.gov-cms
# VACMS-37630: Add support for action links.

## Description

Relates to #[37630](https://github.com/department-of-veterans-affairs/va.gov-team/issues/37630)
related to https://github.com/department-of-veterans-affairs/va.gov-cms/issues/6604

## Testing done
Edited a page and added the action links class. Confirmed on page save the class remains.

## Screenshots


## QA steps
Edit any page with a rich text field, add a link, edit source and add the class for the action links. Save the page and confirm the class appears in the code when inspected. 

As cms user
1. I can edit `/node/33339/edit` 
2. scroll down to a rich text paragraph and click 'source'
3. add the classes `vads-c-action-link--blue` and `vads-c-action-link--green` to two links and 
  - [x] check preview experience for that node
4. trigger a content release and 
  - [x] check the web version of that node.


### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
